Consul/Interfaces/IHealthEndpoint.cs

```diff
@@ -36,7 +36,9 @@ public interface IHealthEndpoint
         Task<QueryResult<ServiceEntry[]>> Service(string service, string tag, CancellationToken ct = default);
         Task<QueryResult<ServiceEntry[]>> Service(string service, string tag, bool passingOnly, CancellationToken ct = default);
         Task<QueryResult<ServiceEntry[]>> Service(string service, string tag, bool passingOnly, QueryOptions q, CancellationToken ct = default);
-        Task<QueryResult<ServiceEntry[]>> Service(string service, string tag, bool passingOnly, QueryOptions q, Filter filter, CancellationToken ct = default);
+        Task<QueryResult<ServiceEntry[]>> Service(string service, string tag, bool passingOnly, QueryOptions q, Filter filter, string healthType, CancellationToken ct = default);
```

Comment:
It is not ok, to change the public method signature in a way that makes it incompatible with previous versions. 
If a new parameter is needed, then the new method overload needs to be added (without modifying the existing one).

Reply:
Alright so, creating leave the method signature as it was and just add a new one with the new parameter.